sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -153,18 +153,22 @@ public static void beforeClass() throws Exception {
 
   private static Cluster buildCluster(CassandraEmbeddedServerBuilder builder) {
     int tried = 0;
-    while (tried < 3) {
+    int delay = 5000;
+    while (tried < 5) {
       try {
         return builder.buildNativeCluster();
       } catch (NoHostAvailableException e) {
```

Comment:
I couldn't find that this exception of type RuntimeException is thrown during cluster startup. We may want to catch all `Exception` here.

Reply:
The Exception is thrown by _ControlConnection.reconnectInternal()_ in the datastax driver component. It is thrown when the client has tried to connect to all newly created cassandra nodes. I'm not sure that all types of exceptions are suited for retrial upon load hence this single catch. WDYT ?

Reply:
Seems fair.